main.tf

```
@@ -47,6 +90,15 @@ resource "netlify_dns_record" "andesite6" {
   value    = hcloud_server.andesite.ipv6_address
 }
 
+resource "netlify_dns_record" "tuta_verifications" {
+  for_each = toset(tutanota_records)
```

```suggestion
  for_each = toset(local.tutanota_records)
```

This won't work as Terraform does not support for_each on a list of objects.